packages/autocomplete-core/src/utils/__tests__/isChrome.test.ts

```diff
@@ -0,0 +1,119 @@
+import { isChrome } from '../isChrome';
+
+describe('isChrome', () => {
+  describe('Android', () => {
+    describe('Mobile/tablet with Chrome', () => {
+      test('returns true with a Samsung S9 (Mobile - Chrome) user agent', () => {
```

Comment:
We know we're on Chrome already.

Same for other tests.

```suggestion
      test('returns true with a Samsung S9 (mobile) user agent', () => {
```